js/src/modal.js

```diff
@@ -184,7 +180,7 @@ class Modal {
     this._setEscapeEvent()
     this._setResizeEvent()
 
-    $(document).off(Event.FOCUSIN, Event.KEYDOWN)
+    $(document).off(`${Event.FOCUSIN} ${Event.KEYDOWN}`)
```

Comment:
Thank you for your fix, but I found that all the events must be specified in the first argument.

And I also added the code in dispose() method.